sentinelsat/sentinel.py

```diff
@@ -1125,7 +1125,7 @@ def _parse_odata_response(product):
         'date': _parse_odata_timestamp(product['ContentDate']['Start']),
         'footprint': _parse_gml_footprint(product["ContentGeometry"]),
         'url': product['__metadata']['media_src'],
-        'Online': product['Online'],
+        'Online': product.get('Online'),
```

Comment:
I wonder what `'Online'` should default to. Now it defaults to `None`. Maybe `True` would be better? Because when you pass the response with `'Online': None` to `api.download`, it will go into the LTA access triggering branch and that will be bad...

So how about `product.get('Online', True)`?

Reply:
Good call, will default to `True`.